### PR TITLE
Update office-365-groups.md

### PR DESCRIPTION
**Pull request by bwinzenz**
Adding important note to Administrator doc to highlight Outlook sync behavior for M365 Groups.
From
https://docs.microsoft.com/en-us/outlook/troubleshoot/user-interface/only-subset-items-synchronized
See the note:
Groups folders only synchronize a maximum of 1 year. Mail to keep offline settings that are lower than 1 year are honored.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you review?

File: microsoft-365/admin/create-groups/office-365-groups.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ The default maximum number of Microsoft 365 groups that an organization can have
 
 Managing your Microsoft 365 groups is more effective when you have actionable information about groups usage. The Microsoft 365 admin center has a reporting tool that lets you see storage use, how many active groups you have, and how users are using the groups. See: [Microsoft 365 Reports in the admin center](../activity-reports/office-365-groups.md) for more information.
 
+>[!IMPORTANT]
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/office-365-groups.md]:
```suggestion
> [!IMPORTANT]
```

File: microsoft-365/admin/create-groups/office-365-groups.md [hunk 2]
```diff
@@ -81,6 +81,9 @@ The default maximum number of Microsoft 365 groups that an organization can have
...
+>Outlook will synchronize a maximum of 12 months of email from a Microsoft 365 Groups folder. If your "Keep Mail Offline" sync slider is set to less than 12 months, Outlook will also honor that setting. The maximum of 12 months cannot be increased. To see email older than 12 months, you must use the Outlook on the Web client.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/office-365-groups.md]:
```suggestion
> Outlook will synchronize a maximum of 12 months of email from a Microsoft 365 Groups folder. If your "Keep Mail Offline" sync slider is set to less than 12 months, Outlook will also honor that setting. The maximum of 12 months cannot be increased. To see email older than 12 months, you must use the Outlook on the Web client.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor formatting.

**Comment by yogkumgit** [comment]
@bwinzenz

**Comment by MikePlumleyMSFT** [comment]
This info is more of concern to Outlook users. I've passed it along to the Outlook writer.

**Pull request closed without merging**